schemas/control-statement-schema.json

```diff
@@ -210,8 +212,13 @@
                   ]
                 }
               }
+            },
+            "source_description": {
```

Comment:
Is it definitely not possible to move this _outside_ the `oneOf` array, into a new `properties` block at same level as the `oneOf`? That would save a lot of repetition, reduce chance of mistakes etc. (If it works, we can also move `exercised_via` there.)